### DISCUSSION TITLE
Fatal error in launcher: Unable to create process using '"d:\bld\spacy_1612273806936\_h_env\python.exe"

### DISCUSSION THREAD

**Discussion by rgoubet** [explosion/spaCy#7359] [Help: Installation] [answered]
On a miniconda installation with spaCy installed from conda-forge, I'm getting the following error message when running `spacy` to download a model:

```
>spacy
Fatal error in launcher: Unable to create process using '"d:\bld\spacy_1612273806936\_h_env\python.exe"  "c:\Users\user\Miniconda3\Scripts\spacy.exe" ': The system cannot find the file specified.
```

I can't reproduce this on other PC's, either Windows or Linux.

**Comment by adrianeboyd** [comment]
Hi, if you can't reproduce it on other systems, it sounds like it might be problem with the conda installation or environment rather than with spacy itself? You can see if calling `python -m spacy` works an alternative or try installing spacy in a new/clean conda env?

**Comment by rgoubet** [comment, marked as answer]
I tried with a new environment, but had the same error. On the other hand, using `python -m spacy` works. I was mostly confused by the reference to the path `"d:\bld\spacy_1612441984352\_h_env\python.exe"`, which doesn't even exist on my system.

**Reply by pl-marasco** [reply]
I know that this is a really old post but for the others that eventually will land here as I did.
 
I had a similar issue with another module (black) under a windows 10 miniconda env. I got a really similar error as you described. 
Fortunatelly I found somewhere that a possible solution for that specific problem can be to install the module once the administrator prompt has been activated. In my case, it has resolved the issue.

**Comment by svlandeg** [comment]
That's definitely weird. But it's good to hear that the workaround with `python -m spacy` works!
